### DISCUSSION TITLE
Estendere i filtri disponibili su "listIssuedDocuments"

### DISCUSSION THREAD

**Discussion by AndreaMorone** [fattureincloud/api#200] [Ideas]
Nello specifico sarebbe interessante poter filtrare "listIssuedDocuments" anche mediante il campo "rc_center" così come avviene per "listReceipts".

**Comment by emajo** [comment]
Ciao @AndreaMorone effettivamente potrebbe essere utile, se tu o qualche altro utente avete altri consigli sui campi che sarebe utile rendere filtrabili possiamo raccoglierli ed eventualmente implementarli dopo averne discusso, sicuramente un altro campo di questo tipo potrebbe essere `e_invoice`.

Grazie,
Emanuele.

**Comment by AndreaMorone** [comment]
Ciao @emajo,
grazie per il followup, in effetti anche `e_invoice` sarebbe molto utile. Provo a fare una analisi interna e suggerisco altri filtri 👍🏻

**Comment by wdalmut** [comment]
anche io mi aggiungo in coda per aggiungere la feature di filtrare per `rc_center` i documenti emessi e ricevuti

**Comment by victoria-vj** [comment]
Anch'io vorrei avere più filtri per "listIssuedDocuments" incluso `subject` oppure `visible_subject` e `rc_center`
Grazie.

**Comment by ivan-bexb** [comment]
Concordo sulle proposte precedenti e aggiungo l'anno della fattura `year` oltre al numero `number` già esistente in modo da poter recuperare una fattura in modo univoco senza post-elaborazione dei risultati.
Comodo ed elegante.
Grazie

**Comment by frabonomi** [comment]
Ciao @emajo, c'è qualche novità sull'aggiunta di filtri? Anche io sarei interessato al filtro in base a `subject` o `visible_subject`. 

Inoltre non riesco a trovare documentazione su quali property è possibile filtrare tramite il query param `q`.

**Reply by valmoz** [reply]
Buongiorno @frabonomi 
la lista dei campi su cui è possibile filtrare è disponibile qui: [Filterable Fields](https://developers.fattureincloud.it/docs/basics/filter-results/queries/#mag-filterable-fields)

Per il resto, purtroppo non ha aggiornamenti da poter fornire.

Grazie
Mauro

**Reply by frabonomi** [reply]
Ah ottimo, grazie per il link! `any_subject` dovrebbe fare al caso mio, mi confermi che serve per filtrare `subject` e `visible_subject` assieme?
